### DISCUSSION TITLE
自动下载的Java下载到与启动器同一文件夹?

### DISCUSSION THREAD

**Discussion by GTEMYee** [Meloong-Git/PCL#2402] [提问] [answered]
如题
可能也就无盘系统用得着?

**Comment by Pigeon0v0** [comment, marked as answer]
%appdata%\.minecraft\cache

**Reply by GTEMYee** [reply]
万一是无盘系统?🤔

**Reply by hejiehao** [reply]
？什么意思

**Reply by GTEMYee** [reply]
学校机房无盘系统，只能U盘留数据，bios也锁了usb引导
而且无盘系统还时不时崩
然而pcl的自动下载Java全凭运气

**Reply by hejiehao** [reply]
你试试手动下载？

**Reply by GTEMYee** [reply]
哪个手动下载，具体表明一下

**Reply by hejiehao** [reply]
有一些发行版提供zip

**Reply by GTEMYee** [reply]
这确实，但是学校网的域名解析烂的就跟shit一样，能不能进去那种网站全靠运气🙄

**Reply by hejiehao** [reply]
从家里提前下载？

**Reply by GTEMYee** [reply]
或者也是，只不过回家时间很少罢了

**Reply by zkitefly** [reply]
https://github.com/Hex-Dragon/PCL2/blob/e6c69750475593f6311c83bb33f0580c072667fa/Plain%20Craft%20Launcher%202/Pages/PageVersion/PageVersionSetup.xaml#L52
